src/components/Contexts/Variables/Modals/OdbImport/OdbImport.tsx

```diff
@@ -46,6 +46,14 @@ export function OdbImport() {
               },
             ],
           },
+          onCompleted(t) {
+            updateConfiguration({
+              variables: {
+                pk: configuration?.pk ?? 1,
+                selectedTarget: t.removeAndCreateBaseTargets[0].pk,
```

Comment:
Is there _always_ at least 1 item in this array?

Reply:
Exactly, at least 1 science target, actually most of the cases it will be just one.
The only cases the UI will receive more than one is when blind offset targets are provided in the observation.

Reply:
In general instruments can have more than one science target

Reply:
Ok, just checking since otherwise we should add a nullish check to the array selection

Reply:
BTW currently navigate is importing targets from the ODB and that should be used just for testing purposes. In the future (not soon), the incoming observations should come from the scheduler instead of being manually imported.